README.md

```
@@ -1,96 +1,273 @@
+
+* Jenkins pipeline for building, testing, and deploying a software project
+* Dockerized Jenkins instance on an AWS EC2 instance
+* EC2 instance as a master node and EC2 instance as a worker node
+* Dockerfile for building the project image
+* Jenkinsfile for configuring the Jenkins pipeline
+
+**Pre-requisites**
+-----------------
+
+Ensure you have the following:
+*AWS Account** with EC2 instances (Ubuntu 20.04 or later)
```

Markdownlint (0.35.0)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)